docs/monitoring-usage.rst

```diff
@@ -278,3 +278,94 @@ follows::
 
 .. _Time Series:
     https://cloud.google.com/monitoring/api/ref_v3/rest/v3/TimeSeries
+
+
+Writing Custom Metrics
+---------------------------
+
+The Stackdriver Monitoring API can be used to write data points to custom metrics. Please refer to
+the documentation on `Custom Metrics`_ for more information.
+
+To write a data point to a custom metric, you must provide an instance of
+:class:`~gcloud.monitoring.metric.Metric` specifying the metric type as well as the values for
+the metric labels. You will need to have either created the metric descriptor earlier ( see the
+`Metric Descriptors`_ section ) or rely on metric type auto-creation, see `Auto-creation of
```

Comment:
whitespace is fixed

Agreed on the comma, I replaced it with another set of parentheses for consistency with the first part of the sentence. The parentheses I'm also iffy about, we could maybe just split it into separate sentences.